repos/system_upgrade/el9toel10/actors/check_default_initramfs/actor.py

```diff
@@ -0,0 +1,22 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import check_default_initramfs as check_default_initramfs_lib
+from leapp.models import DefaultInitramfsInfo
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckDefaultInitramfs(Actor):
+    """
+    Checks whether the default initramfs uses problematic dracut modules.
+
+    Checks whether dracut modules that are missing on the target system are used.
```

Comment:
```suggestion
    Checks whether dracut modules that are known to be missing on the target system are used.
```
Not a blocker, just when I first read that I thought it's going to be automatically detected and not hardcoded.

Reply:
It theory, it could be done, however, the implementation would be of a much larger scope than the current one. This PR already goes way out of the scope of the ticket and its proposed solution (checking that a certain file exists). This needs to be discussed with PO/tech lead, to know whether we can burn dev time on such a broad implementation.